dwbb/src/interface/DW_asymfifo_s2_sf.scala

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024 Jiuyang Liu <liu@jiuyang.me>
+package org.chipsalliance.dwbb.interface.DW_asymfifo_s2_sf
+
+import chisel3._
+import chisel3.experimental.SerializableModuleParameter
+import upickle.default
+
+object Parameter {
+  implicit def rw: default.ReadWriter[Parameter] =
+    upickle.default.macroRW[Parameter]
+}
+
+case class Parameter(
+    dataInWidth: Int = 16,
+    dataOutWidth: Int = 8,
+    depth: Int = 16,
+    pushAeLvl: Int = 2,
+    pushAfLvl: Int = 2,
+    popAeLvl: Int = 2,
+    popAfLvl: Int = 2,
+    errMode: Boolean = false,
```

Comment:
same here.

Reply:
Using `String`s instead of `Boolean`s in the revised code.